### DISCUSSION TITLE
Play in french

### DISCUSSION THREAD

**Discussion by Xenders59** [tsunamods-codes/7th-Heaven#126] [Q&A] [answered]
Hello , i have the steam game  and i use the game convertor ,its work  but its impossible to play in french  its start in english .
Iv try to apply french retranslate mod , it work when i have no mod, its in french  so i have installed all the tsunamods  and the game came back in english , iv try to reinstall the french patch and this time its not work

I cant find option  to change it , could you help?

Thanks

**Comment by julianxhokaxhiu** [comment, marked as answer]
Hi, 7th Heaven is meant to be used on top of the English Game only. Any other language is not supported. If there's a re-translation pack for 7th Heaven, please consider asking for help on their relative support channels.
